### PR TITLE
Tidy files pt. 2

### LINKED ISSUE #1930: Helper/util file tidying following https://github.com/bbc/simorgh/issues/1535
**Is your feature request related to a problem? Please describe.**
There are some more helpers left in app that should be in utils. 

**Describe the solution you'd like**
1. Move everything (except the readme which should be deleted) in
src/app/helpers
to:
src/app/lib/utilities

2. Move contents of  src/lib/json into src/lib/utilities
3. Move src/lib/blockHandlers.js into src/lib/utilities (also please double check contents as I think the second export is never used)
4. Add to the readme in src/lib/ that the utilities/ directory is for helpers/utils that are used in production. And be clear that testHelpers/ is where we store non-production helpers/utils. Also clarify that having helpers/utils next to their components/containers is fine, but as soon as they're used beyond that component/container they must move to either of the main utils or helpers directories (depending on whether they're for tests or prod code), also reaffirm that we don't mind whether they're called utils or helpers when within a component/container folder (similar wording to what we have now).

**Describe alternatives you've considered**
None, this is a basic standardisation of where helpers/utils live.

**Testing notes**
[Tester to complete]

Dev insight: Regression required

**Additional context**
Follows but not blocked by: https://github.com/bbc/simorgh/issues/1535

### PR DESCRIPTION
**Pull request by 12**
Resolves #1930 & #1535 

**Overall change:** 
Update file structure with regards to helpers and utilities to maintain consistency

**Code changes:**

- Move helpers used in production to `/src/app/lib/utilities`
- Move helpers not used in production to `/src/testHelpers`
- Update readme to document how helpers and utils should be split and folder-ed

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] Tests added for new features
- [ ] Test engineer approval
- [ ] I have followed [the merging checklist](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/MERGE_PROCESS.md) and this is ready to merge.

### CODE REVIEW

**Review by sareh — APPROVED** [review]
👍 Looks good!

**Review by sareh — CHANGES_REQUESTED** [review]
There are some imports that need updating. Please check these with `npm run test:lint`.

```
simorgh/src/app/helpers/pageAnchor.js
  1:22  error  Unable to resolve path to module './onClient'  import/no-unresolved

simorgh/src/app/helpers/pageAnchor.test.js
  2:22  error  Unable to resolve path to module './onClient'  import/no-unresolved

simorgh/src/app/lib/utilities/preprocessor/rules/blockPositioning.js
  1:21  error  Unable to resolve path to module '../../../json/deepGet'    import/no-unresolved
  2:23  error  Unable to resolve path to module '../../../json/deepClone'  import/no-unresolved

simorgh/src/app/lib/utilities/preprocessor/rules/blockPositioning.test.js
  1:23  error  Unable to resolve path to module '../../../json/deepClone'  import/no-unresolved
```

**Review by sareh — DISMISSED** [review]
Internet was slow, so didn't see the fix pushed before my comment. Fix looks good, thanks @rhell

**Comment by dr3** [comment]
Personaly preference, we have alot of files in these folders, rather than 
```
utils
- deepClone.js
- deepClone.test.js
```
can we do 
```
utils
- deepClone
-- index.js
-- index.test.js
```
Would be nice to do this across simorgh IMO

File: src/app/lib/README.md [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 This directory contains core files and references used across the application.
 
+The `./utilities` directory is where helpers and utiltiies used in production should be maintained, whereas non-production helpers and utilities should be stored in `../testHelpers`. Helpers nested within their containers or components is permitted, on the basis that that container or component is the only thing using said helper. If the helper needs to be used elsewhere, it should be moved to within its respective global-scope folder (`utilities` or `testHelpers`). The terms `utiltiies` and `helpers` are interchangable.
```
Comment (jamesbhobbs) [inline src/app/lib/README.md:5]: Nicely written

**Review by jamesbhobbs — APPROVED** [review]

**Review by dr3 — APPROVED** [review]

**Review by sareh — APPROVED** [review]

**Comment by PriyaKR** [comment]
Regression looks good to me.

**Pull request merged**